### ISSUE TITLE
Outdated Macedonia translations

### ISSUE THREAD

**Issue by cilim** [countries/countries#624]
Hi there 👋

> In June 2018, Macedonia and Greece resolved the conflict with an agreement that the country should rename itself Republic of North Macedonia. This renaming came into effect in February 2019.

ref: https://en.wikipedia.org/wiki/North_Macedonia

It would be nice if this was updated here too, so we get:

```
ISO3166::Country.find_country_by_alpha2("mk").translations['en']
=> "North Macedonia"
```

The current result is `Macedonia, Republic of` ❌

**Comment by rgardien** [comment]
This will be solved by bumping the i18n_data dependency to 0.13.0 instead of 0.10.0.

Anyone can pick this up please? 👍

**Comment by jana4u** [comment]
I would relax dependency more to `gem.add_dependency('i18n_data', '~> 0.10')`. Otherwise we have the same issue once 0.14 is released.

**Comment by cilim** [comment]
> I would relax dependency more to gem.add_dependency('i18n_data', '~> 0.10')

Agreed.

@rgardien if that is all that is needed, I can open up a PR for you if that's ok with you? :)

**Comment by pmor** [comment]
This has been fixed, current 'i18n_data' dependency is '~> 0.15.0'
